Components/grid/grid.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+
+/* colors for unit borders */
+:root {
+    --color1: #001219;
+    --color2: #005f73;
+    --color3: #3d9397;
+    --color4: #94d28d;
+    --color5: #e9d8a6;
+    --color6: #ee9800;
+    --color7: #ca6702;
+    --color8: #bb3e03;
+    --color9: #ae2012;
+    --color10: #9b2226;
+    --color11: rgba(255, 255, 255, 0.8);
+}
+
+/* general web-page styles */
+#collections {
+    margin: 0 auto;
+    padding: 0 var(--space-md);
+}
+
+#collections ul{
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+#collections h2 {
+    font-size: calc(2.5 * var(--paragraph));;
```

Comment:
I am not able to understand what `--paragraph` variable is for? So I think you could give it a better name according to the purpose it is serving. If it's for font-size then you could use something like `--font-size-para` or `--fs-paragraph`

Reply:
please visit typography.css you will find all the typography-related CSS variables there and the paragraph CSS variable is for any HTML texts element user can use to maintain the same font size of the declared HTML. yes it will be better to mention that it is for maintaining font-size of paragraph I will make the changes thanks 👍🏼